spec/models/audits/response_spec.rb

```diff
@@ -1,4 +1,4 @@
-RSpec.describe Response do
+RSpec.describe Audits::Response do
```

Comment:
If you enclose the test within `module Audits` there is no need to scope every class inside. Same comment applies to remaining RSpec tests.